doc/changes/latest.inc

```diff
@@ -34,6 +34,7 @@ Bugs
 - Fix bug with PySide6 compatibility (:gh:`11721` by `Eric Larson`_)
 - Fix hanging interpreter with matplotlib figures using ``mne/viz/_mpl_figure.py`` in spyder console and jupyter notebooks`(:gh:`11696` by `Mathieu Scheltienne`_)
 - Fix bug with overlapping text for :meth:`mne.Evoked.plot` (:gh:`11698` by `Alex Rockhill`_)
+- Deprecated ``gap_description`` kwarg of :func:`mne.io.read_raw_eyelink` and changed default value to ``'BAD_ACQ_SKIP'`` (:gh:`11719` by `Scott Huberty`_)
```

Comment:
This should go under the API changes section
```suggestion
- Deprecated ``gap_description`` keyword argument of :func:`mne.io.read_raw_eyelink` and changed default value to ``'BAD_ACQ_SKIP'`` (:gh:`11719` by `Scott Huberty`_)
```

Reply:
Thanks @hoechenberger  I added the deprecation announcement to the API changes section.